tests/traits/ImportDatasetTrait.php

```diff
@@ -34,6 +34,8 @@ public static function importDatatasetNoAuthDbSsl(string $collection, string $da
             '--host',
             'mongodb-ssl',
             '--ssl',
+            '--sslCAFile=/tmp/ca-cert.pem',
+            '--sslPEMKeyFile=/tmp/client-cert-and-key.pem',
```

Comment:
Na vsetkych miestach chybalo nastavenie SSL parametrov.
Mongo DB potrebuje `cert + #key` ako jeden subor (je to samostatny commit dalej).

-----

Na tomto mieste chybali parametre pri importe dat, pred `DataDir` testom.